tasks/ci.rake

```diff
@@ -182,6 +182,17 @@ EOS
       beaker_suite(:aio)
     end
 
+    desc <<-EOS
+Run the acceptance tests using puppet-agent (AIO) packages with retries.
+
+  $ SHA=<full sha> bundle exec rake ci:test:aioretry
+
+SHA should be the full SHA for the puppet-agent package.
+EOS
+    task :aioretry => ['ci:check_env', 'ci:gen_hosts'] do
```

Comment:
I'd call this `:aio_retry` for readability

Reply:
Done.

Reply:
Removed aio_retry as a separate task and now use a rake task parameter for :aio instead.